app/views/candidate_mailer/new_referee_request.text.erb

```diff
@@ -2,8 +2,8 @@ Dear <%= @candidate_name %>,
 
 # Give details of a new referee
 
-<%= t('new_referee_request.not_responded.explanation', referee_name: @referee_name) %>
+<%= @explanation %>
 
-Reply to this email with the name and email address of a new referee, and tell us how you know them.
+<%= @reply %>
```

Comment:
Can `@explanation` and `@reply` be merged? 

Also, I think it's cleaner to call the `t()` code inside the view rather than the mailer class. WDYT?

Reply:
Yes to both! Definitely cleaner.

Reply:
Refactored now, some much better now. Thank you!